### DISCUSSION TITLE
Podman Tools for UniFi Cloud Key Gen 2 Running UniFiOS

### DISCUSSION THREAD

**Discussion by charlestephen85** [unifi-utilities/unifi-common#547] [Ideas]
I see the tools for UDM etc, but can't seem to find the same tools for CloudKey Gen 2 Pro. I have a CloudKey with 1TB of space that I use to run UniFi Network but I have a separate NVR for UniFi Protect and would love to run a logging/monitoring server on the CloudKey. Do the Podman configurations work natively given that it's running UniFiOS or would it take some special sauce?

**Comment by boostchicken** [comment]
I dont think it ever worked with a Cloud Key. If podman is there give it a try.
